docs/source/upcoming_release_notes/614-LAMP_motion_system_class.rst

```diff
@@ -0,0 +1,33 @@
+614 LAMP motion system class
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- LAMP Class: TMO-LAMP Motion system class:
+ - Gas Jet X/Y/Z Axes
+ - Gas Needle X/Y/Z Axes
+ - Sample Paddle X/Y/Z Axes 
```

Comment:
Suggesting adding more context in case someone doesn't know that lamp isn't a light you put on your desk. Also some formatting.
```suggestion
- LAMP motion Class for the LAMP endstation in TMO. This includes the following motion axes:
  - Gas Jet X/Y/Z
  - Gas Needle X/Y/Z
  - Sample Paddle X/Y/Z
```

Reply:
Resolved in commit 461e9d4